post.css

```diff
@@ -487,4 +487,275 @@ select {
+    }
+
+    .rectangle {
+        padding-top: 52px;
+        width: 854px;
+    }
+
+    .rectangle_caption {
+        font-size: 16px;
+        max-width: 481px;
+    }
+
+    .small_text_about_blog {
+        font-size: 20px;
+        max-width: 550px;
+        padding-top: 32px;
+    }
+
+    .text_of_section {
+        font-size: 20px;
+    }
+
+    .pipeline {
+        font-size: 32px;
+    }
+
+    .top_photo {
+        padding-top: 52px;
+    }
+
+    .top_padding_burger {
+        padding: 35px 30px;
+    }
+
+    .list,
+    .text_after_list,
+    .end {
+        display: block;
+        padding-top: 52px;
+    }
+
+    .share,
+    .phone {
+        display: none;
+    }
+
+    .list_part {
+        padding-top: 20px;
+    }
+
+    .share_on {
+        display: block;
+        font-family: 'SF Pro Display';
+        font-weight: 500;
+        font-size: 16px;
+        line-height: 160%;
+        white-space: nowrap;
+        color: #000000;
+        margin-top: -4px;
+    }
+
+    .twitter2 {
+        border-right: 0;
+        padding-right: 80px;
+        padding-left: 80px;
+    }
+
+    .facebook2 {
+        padding-right: 71px;
+        padding-left: 71px;
+    }
+
+    .table_of_contact2 {
+        width: 600px;
+        display: flex;
+        justify-content: space-between;
+    }
+
+    .table_footer {
+        margin: 0;
+    }
+
+    .section_footer_text {
+        font-size: 16px;
+        padding-bottom: 34px;
+    }
+
+    .comment_from_author {
+        padding-top: 32px;
+    }
+
+    .big_avatar_text_bold {
+        font-size: 18px;
+    }
+
+    .big_avatar {
+        padding-top: 4px;
+        padding-right: 24px;
+
+    }
+
+    .big_avatar_text {
+        font-size: 16px;
+        padding-top: 0px;
+    }
+
+    .read_next {
+        font-size: 44px;
+        padding-top: 73px;
+    }
+
+    .additional_articles {
+        margin-top: 132px;
+    }
+
+    .sign_up_caption {
+        padding-top: 62px;
+        font-size: 36px;
+    }
+
+    .sign_up_text {
+        font-size: 20px;
+        padding-top: 18px;
+    }
+
+    .input_mail {
+        height: 55px;
+        width: 400px;
+    }
+
+    .mail {
+        padding: 36px 40px 60px 40px;
+    }
+
+    ::placeholder {
+        font-size: 20px;
+        padding-left: 16px;
+    }
+
+    .mail_text {
+        height: 55px;
+        width: 120px;
+        padding-top: 14px;
+        font-size: 20px;
+    }
+
+    .square_first_text {
+        left: 50%;
+        transform: translateX(-50.0%);
+    }
+
+    .scroll_text {
+        font-size: 22px;
+        letter-spacing: 0.1em;
+    }
+
+    .lorem {
+        font-size: 16px;
+        max-width: 488px;
+    }
+
+    .link {
+        font-size: 16px;
+    }
+}
+
+@media (min-width:1024px) {
+    .desktop_menu {
+        display: flex;
+        gap: 32px;
+        font-family: 'New York Small';
+        font-weight: 500;
+        font-size: 20px;
+        line-height: 170%;
+        letter-spacing: 0.12em;
```

Comment:
I recommend you not to use `em`. You can do your own research and decide for yourself, but the main problem with it is that it depends on the font size of the parent element. so when that changes it will also affect current element. you can check `rem` instead. there are also other places where you used `em`. this isn't a required change, just want to make sure that you understand the things that you use